listenbrainz/webserver/views/api.py

```diff
@@ -635,6 +636,34 @@ def get_playlists_collaborated_on_for_user(playlist_user_name):
     return jsonify(serialize_playlists(playlists, playlist_count, count, offset))
 
 
+@api_bp.route("/user/<user_name>/services", methods=['GET', 'OPTIONS'])
+@crossdomain
+@ratelimit()
+def get_service_details(user_name):
+    """
+    Get list of services which are connected to a given user's account.
+
+    .. code-block:: json
+
+        {
+            "user_name": "hwnrwx",
+            "services": ["spotify"]
+        }
+
+    :param user_name: the MusicBrainz ID of the user whose similar users are being requested.
+    :resheader Content-Type: *application/json*
+    :statuscode 200: Yay, you have data!
+    :statuscode 403: Forbidden, you do not have permissions to view this user's information.
```

Comment:
https://github.com/metabrainz/listenbrainz-server/blob/b28e14f2843f221cd5cbe555c9eabdc50c1a6e36/listenbrainz/webserver/views/user_timeline_event_api.py#L253

saw this from the link above
But we should give 403 on wrong token but you guys are giving 401.

Reply:
Ah. Good catch. I'll open a ticket for fixing this.